web/src/components/CityContentFooter.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { DISCLAIMER_ROUTE, LICENSES_ROUTE, pathnameFromRouteInformation } from 'api-client'
+
+import buildConfig from '../constants/buildConfig'
+import CleanLink from './CleanLink'
+import Footer from './Footer'
+
+const SidebarFooterContainer = styled.div`
+  width: 100%;
+  margin: -10px 12px 0 23px;
+  > * {
+    color: ${props => props.theme.colors.textColor};
+    height: 48px;
```

Comment:
```suggestion
    padding: 16px 0;
```
setting fixed heights reduces the flexibility of a layout. I prefer using a padding here